thirdparty/DiskANN/src/pq_flash_index.cpp

```diff
@@ -825,7 +826,7 @@ namespace diskann {
                                            const _u64  beam_width,
                                            const bool  use_reorder_data,
                                            QueryStats *stats,
-                                           faiss::BitsetView bitset_view) {
+                                           faiss::BitsetView bitset_view, bool smaple=false) {
```

Comment:
Typo. and can you explain this flag? Also the default value should be in .h file right?

Reply:
> Typo. and can you explain this flag? Also the default value should be in .h file right?

sorry forgot this :)